### ISSUE TITLE
Exported `cli` projects should use a default template. Instead of using vercel as a default template

### ISSUE THREAD

**Issue by JayaKrishnaNamburu** [webstudio-is/webstudio#2638]
Exported `cli` projects currently use `vercel` as default template. So the default files from the `templates/default` are merged with `templates/vercel` if no template attribute is passed.
https://github.com/webstudio-is/webstudio/blob/main/packages/cli/src/commands/build.ts#L23

But it's creating issues for users who use the `webustudio build` command. And try to export them to `netlify` or some custom sources. Because it's adding default `vercel` files and configurations. 

```js
export const buildOptions = (yargs: CommonYargsArgv) =>
  yargs
    .option("assets", {
      type: "boolean",
      default: true,
      describe: "[Experimental] Download assets",
    })
    .option("preview", {
      type: "boolean",
      default: false,
      describe: "[Experimental] Use preview version of the project",
    })
    .option("template", {
      type: "string",
      default: "vercel",
      describe: `[Experimental] Template to use for the build [choices: ${PROJECT_TEMPALTES.toString()}]`,
    });
```

Related issue #2615

**Comment by kof** [comment]
@istarkov we need to remove default value to be set to anything at all and if its not set - throw an error that explains what to do.

Init will ask user and set it, its only a problem when individual commands are used.

@JayaKrishnaNamburu please change this any time you can

**Comment by istarkov** [comment]
Just check that in all existing fixtures only cli command will be changed. Files etc should be the same

**Comment by istarkov** [comment]
Anyway the issue that even non vercel is set as template, vercel template is used. Thats wrong

**Comment by kof** [comment]
Yes, we have no otherw way, because if vercel template is used implicitly, we don't know if that's intentional or not, we can't throw an error when user decides to use template X and deploy to platform Y.

We just need to make sure they explicitly used that template.

**Comment by JayaKrishnaNamburu** [comment]
Fixed in #2660
